pom.xml

```diff
@@ -69,20 +69,22 @@
 			<version>9.4-1204-jdbc42</version>
 		</dependency>
 
-
 		<dependency>
 			<groupId>org.springframework</groupId>
 			<artifactId>spring-context</artifactId>
 			<version>4.2.1.RELEASE</version>
 		</dependency>
-
 		<dependency>
 			<groupId>org.springframework</groupId>
 			<artifactId>spring-orm</artifactId>
 			<version>4.2.1.RELEASE</version>
 		</dependency>
 
-
+		<dependency>
+			<groupId>javax.mail</groupId>
+			<artifactId>mail</artifactId>
+			<version>1.4</version>
```

Comment:
I see that this is not the last version. Wouldn't it be nice to use the most recent stable version? 1.4.7